output/opendap_post2.sh

```diff
@@ -24,6 +24,7 @@ THIS=$(basename -- $0)
 #
 EXIT_SUCCESS=0
 EXIT_ERROR=1
+sendEmail="no"
```

Comment:
I have to ask because it's not obvious to me, but is this necessary and does it simplify notifications or make them more complicated?

Reply:
Thank you for going over this. :-) Yes I think it is necessary. This is a simplification that harmonizes the periodic posting of status files as well as one-time posting of result files at the completion of a model run. It also covers the cases where the email address list in `OPENDAPNOTIFY` is unset or set to `null`, and whether or not the `sendNotification` tracer was found in the list of files to post, whether in the middle or at the end, or if it was never found at all.  It also covers the case where the `asgs-sendmail` command failed.